### PR TITLE
Add some defines

### PR DESCRIPTION
**Pull request by ASDosjani**

### CODE REVIEW

File: configs/default/SJET-AOCODARCH7DUAL.config [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Betaflight / STM32H743 (SH74) 4.3.0 Feb  1 2022 / 20:15:46 (3267f0417) MSP API: 1.44
 
+#define USE_ACCGYRO_BMI270
+#define USE_BARO_DPS310
+#define USE_BARO_DPS310
```
Comment (haslinghuis) [inline configs/default/SJET-AOCODARCH7DUAL.config:5]: duplicate

Comment (haslinghuis) [reply configs/default/SJET-AOCODARCH7DUAL.config:5]: Please provide link to board information for verification

Comment (ASDosjani) [reply configs/default/SJET-AOCODARCH7DUAL.config]: Yes, I noticed but thanks.

**Review by haslinghuis — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Comment by blckmn** [comment]
Will merge after the installation of the PR for github actions. Will be used as a test :)

**Pull request merged**